README.md

```diff
@@ -20,6 +20,9 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 
 ## Release Notes
 
+2.0.25:
```

Comment:
Should probably set this version number to `2.0.27`
There will be at least two "releases" - the preview one and the standard release, which can't share version numbers between the two types

Reply:
will use the placeholder NEXTRELEASE for this and allow it to be specified during our regular release process.